specification/frontdoor/resource-manager/readme.md

```diff
@@ -28,9 +28,27 @@ These are the global settings for the FrontDoor API.
 title: FrontDoorManagementClient
 description: FrontDoor Client
 openapi-type: arm
-tag: package-2019-11
+tag: package-2020-01
 ```
 
+### Tag: package-2020-01
+
+These settings apply only when `--tag=package-2020-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-01'
+input-file:
+- Microsoft.Network/stable/2019-11-01/network.json
```

Comment:
>/2019-11-01 [](start = 26, length = 11)

can update this to 2020-01-01 as well, since we are copying network.json

Reply:
Updated

---
In reply to: [365443052](https://github.com/Azure/azure-rest-api-specs/pull/8115#discussion_r365443052) [](ancestors = 365443052)